src/workflows-app/SubmissionConfig.js

```diff
@@ -479,7 +491,6 @@ export const BaseSubmissionConfig = (
         div(
           {
             style: {
-              backgroundColor: 'rgb(235, 236, 238)',
```

Comment:
I saw that this change was made for `a11y` compatibility. Since I'll probably have to do the same: how did you discover that this line was a problem?

Reply:
Moved to Slack